images/kubekins-e2e/Dockerfile

```diff
@@ -93,9 +93,11 @@ RUN if [ -n "${KIND_VERSION}" ]; then \
     fi
 
 # install kubetest2 binaries if a version is provided
+# kubetest2 must be installed from git as some important build flags are set by make
```

Comment:
what importan build flags?

that is the problem with `go install`?

Reply:
yes, the makefile embeds the git version and the day it was built.

Reply:
then can you make this explicit
```suggestion
# kubetest2 must be installed from git as the makefile embeds the git short hash and the day it was built.
```